INSTALL.rst

```diff
@@ -5,6 +5,6 @@ Invenio-Access is on PyPI so all you need is:
 
 .. code-block:: console
 
-   $ pip install Invenio-Access
+   $ pip install --pre invenio-access
```

Comment:
``--pre`` should not be included. Eventually it will work and then it will be a lot of work to go back and remove the ``--pre`` again.